### DISCUSSION TITLE
Terms & Conditions

### DISCUSSION THREAD

**Discussion by Multiverse76** [DaxStudio/DaxStudio#781] [General]
I'm having problems getting Dax Studio approved by IT as there is no mention of how the program / organisation handles data in relation to GDPR etc.  They are suspecting the software is paid for by our data.  Is there a page / resource available that addresses these concerns so that I can get IT to approve the software?

Thanks ever so!

**Comment by marcosqlbi** [comment]
DAX Studio is free open-source and funded by sponsors and donations.
You should contribute to it and review the source code to make sure that no data is used!
https://github.com/sponsors/DaxStudio

**Comment by dgosbell** [comment]
The reason there is no statement about GDPR or how we handle and organize data is because we do not store or handle **any** user data. DAX Studio is a query tool. It connects to a tabular data store like Power BI or Analysis Services and lets you run queries and see the results of queries you run. The data that is returned in response to the queries that you generate travels from the data model to your machine that is running DAX Studio, it never goes via any other intermediate service that is controlled by us.
